routes/routes.js

```diff
@@ -0,0 +1,67 @@
+const express = require('express');
+const { keysToCamel } = require('../common/utils');
+
+const router = express.Router();
+const {db} = require('../server/db');
+
+router.use(express.json());
+
+router.post('/', async (req, res) => {
+  try {
+    const data = req.body;
+    const newRoute = await db.query(
+      `INSERT INTO routes (${data.driverId ? 'driver_id, ' : ''} id) 
+      VALUES(${data.driverId ? `${data.driverId}, ` : ''} ${data.id}) RETURNING *;`,
+    );
+    res.status(200).json(keysToCamel(newRoute[0]));
+  } catch (err) {
+    res.status(500).send(err.message);
+  }
+});
+
+router.get('/', async (req, res) => {
+  try {
+    const allRoutes = await db.query(`SELECT * FROM routes;`);
+    res.status(200).json(keysToCamel(allRoutes));
+  } catch (err) {
+    res.status(500).send(err.message);
+  }
+});
+
+router.get('/:routeId', async (req, res) => {
+  try {
+    const { routeId } = req.params;
+    const routeInfo = await db.query(`SELECT * FROM routes WHERE id = $1;`, [routeId]);
```

Comment:
since we're now using `db` instead of `pool`, we can now specify named parameters in the query, for example instead of `(...'...WHERE id = $1;', [routeId]);` you can replace it with `(...'...WHERE id = $(routeId);', { routeId });`

Reply:
pls do the same with any other unnamed parameters in a query (ie `$1`, `$2`, etc)